### ISSUE TITLE
Wonder about lrn parameters.

### ISSUE THREAD

**Issue by enterAI** [dontfollowmeimcrazy/imagenet#2]
Hi.
I am curious about the 'local response normalization' layer parameters you set.
The paper case is a little bit different from what you set except 'beta'.

What does it mean to be different from the actual setting?

**Comment by dontfollowmeimcrazy** [comment]
Hello @enterAI,

for the hyperparameters I just followed the working pre-trained implementation [here](http://www.cs.toronto.edu/~guerzhoy/tf_alexnet/). Did not have the chance to test other parameter setups. If you have the opportunity than let me know the results 👍🏻.
